crates/modelardb_storage/src/parser.rs

```diff
@@ -476,6 +485,49 @@ impl ModelarDbDialect {
 
         Ok(statement)
     }
+
+    /// Return [`true`] if the token stream starts with VACUUM, otherwise [`false`] is returned.
+    /// The method does not consume tokens.
+    fn next_token_is_vacuum(&self, parser: &Parser) -> bool {
+        // VACUUM.
+        if let Token::Word(word) = parser.peek_nth_token(0).token {
+            word.keyword == Keyword::VACUUM
+        } else {
+            false
+        }
+    }
+
+    /// Parse VACUUM \[table_name\[, table_name\]+\] to a [`Statement::ShowVariable`] with the
+    /// table names in the `variable` field. Note that [`Statement::ShowVariable`] is used since
+    /// [`Statement`] does not have a `Vacuum` variant. A [`ParserError`] is returned if VACUUM is
```

Comment:
maybe its just me who don't get it, but what does "VACUUM is typed incorrectly" mean?